workflow-ui/compose/src/androidTest/java/com/squareup/workflow1/ui/compose/ComposeViewFactoryTest.kt

```diff
@@ -36,6 +37,7 @@ internal class ComposeViewFactoryTest {
   @get:Rule val rules: RuleChain =
     RuleChain.outerRule(DetectLeaksAfterTestSuccess())
       .around(IdleAfterTestRule)
+      .around(IdlingDispatcherRule)
```

Comment:
Probably time to write a `WatchLeaksAndIdleBetweenDispatches<R: Rule>`, where `R` would give us access to the wrapped `ComposeRule` or `ScenarioRule`.

Reply:
The order's different because...  I honestly don't know?  But nice catch -- I'll fix.

I agree that a composite rule seems to make sense.  Once this is merged, I can address the leaks found by the LeakCanary rule changes and combine all the logic.